staff/aleix-caminal/app-template/about.html

```diff
@@ -27,8 +37,5 @@ <h2>Some of our clients</h2>
                 </ul>
```

Comment:
forgot to delete it maybe

Reply:
the missing part is inside a collapsible section in github, may you have missed it